src/screens/farmProfile/FarmProfile.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import FarmContactCard from './components/FarmContactCard';
+import leftArrow from '../../assets/left_arrow.png';
+import '../../styles/FarmProfile.css';
+
+class FarmProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null
+    };
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
```

Comment:
This is extracting the farmID from the route URL right? Maybe rename it unless if this is convention.